src/main/java/org/aion/wallet/ui/components/HistoryPane.java

```diff
@@ -0,0 +1,93 @@
+package org.aion.wallet.ui.components;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.collections.FXCollections;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import org.aion.base.type.Address;
+import org.aion.base.util.ByteUtil;
+import org.aion.base.util.TypeConverter;
+import org.aion.wallet.connector.BlockchainConnector;
+import org.aion.wallet.connector.WalletBlockchainConnector;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.math.RoundingMode;
+import java.net.URL;
+import java.util.List;
+import java.util.ResourceBundle;
+import java.util.stream.Collectors;
+
+import static org.aion.wallet.util.BalanceFormatter.WEI_MULTIPLIER;
+
+public class HistoryPane implements Initializable {
+
+    private final BlockchainConnector blockchainConnector = new WalletBlockchainConnector();
+    @FXML
+    private TableView<TxRow> txListOverview;
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+        TableColumn typeCol = new TableColumn("Type");
```

Comment:
Can be generified: 
`TableColumn<TxRow, String> typeCol = new TableColumn<>("Type");`